scripts/docker/gatkbase/Dockerfile

```diff
@@ -6,7 +6,7 @@ FROM ubuntu:18.04
 RUN apt-get update && \
```

Comment:
OpenJDK 8 reference on line 1 but GitHub won't let me comment there

Reply:
@mcovarr Thank you, good catch!